### PR TITLE
[depends] Update libusb to v1.0.22

### PR DESCRIPTION
**Pull request by garbear**
Compile-tested after merging https://github.com/kodi-game/peripheral.steamcontroller/pull/6.

### CODE REVIEW

**Comment by AlwinEsch** [comment]
@garbear OK to bring in?

File: depends/common/libusb/CMakeLists.txt [hunk 1]
```diff
@@ -6,10 +6,12 @@ include(ExternalProject)
 ExternalProject_Add(
     libusb
     SOURCE_DIR          ${CMAKE_SOURCE_DIR}
-    CONFIGURE_COMMAND   <SOURCE_DIR>/autogen.sh
-                            --prefix=${OUTPUT_DIR}
+    CONFIGURE_COMMAND   <SOURCE_DIR>/bootstrap.sh
+    COMMAND             <SOURCE_DIR>/configure
```
Comment (Rechi) [inline depends/common/libusb/CMakeLists.txt:10]: Why is the switch from `autogen.sh` to `bootstrap.sh` and `configure` required?

File: depends/common/libusb/CMakeLists.txt [hunk 2]
```diff
@@ -6,10 +6,12 @@ include(ExternalProject)
...
+                            --prefix=${CMAKE_INSTALL_PREFIX}
```
Comment (Rechi) [inline depends/common/libusb/CMakeLists.txt:11]: Changing `prefix` from `${OUTPUT_DIR}` to `${CMAKE_INSTALL_PREFIX}` doesn't look right.

**Comment by garbear** [comment]
The add-on doesn't work, and there is support at the OS level for some distros now, so I don't have any thoughts on if this is merged.

**Comment by garbear** [comment]
Coming in with the dinosaur merge!

**Pull request merged**